pdq/DMS.go

```diff
@@ -0,0 +1,18 @@
+package pdq
+
+import (
+	"fmt"
+	"math"
+)
+
+func CalcDMS(abstractness float64, instability float64) (float64, error) {
+	if abstractness < 0 || instability < 0 {
+		return -1, fmt.Errorf("invalid metric value: abstractness %v, instability %v; values must be 0 or higher", abstractness, instability)
+	}
+
+	if abstractness >= 1e308 || instability >= 1e308 {
+		return -1, fmt.Errorf("invalid metric value: abstractness %v, instability %v; values must be lesser than infinity", abstractness, instability)
```

Comment:
invalid metric value: abstractness %v, instability %v; values must be 0 or higher

do we need this part i**nvalid metric value**.... probably should be obvious that  if u got error from metric calc func ...means smth is wrong could be like 

abstractness %v, instability %v; values must be >= than 0